src/components/cc-logs-instances/cc-logs-instances.types.d.ts

```diff
@@ -0,0 +1,44 @@
+
+export type LogsInstancesState = LogsInstancesStateLoading | LogsInstancesStateError | LogsInstancesStateLoaded;
+
+export interface LogsInstancesStateLoading {
+  state: "loading";
+}
+
+export interface LogsInstancesStateError {
+  state: "error";
+}
+
+export interface LogsInstancesStateLoaded {
+  state: "loaded";
+  mode: "live" | "cold";
```

Comment:
question: In the rest of the file, the unions are in a dedicated type, why not here?